charts/netbird/templates/management-deployment.yaml

```diff
@@ -69,10 +69,10 @@ spec:
           {{- end }}
           ports:
             - name: {{ .Values.management.service.name }}
-              containerPort: {{ .Values.management.service.port }}
+              containerPort: {{ .Values.management.service.containerPort }}
               protocol: TCP
             - name: {{ .Values.management.serviceGrpc.name }}
-              containerPort: {{ .Values.management.serviceGrpc.port }}
+              containerPort: {{ .Values.management.serviceGrpc.containerPort }}
```

Comment:
I would use values.management.gprcContainerport instead of .Values.management.serviceGrpc.containerPort